Level-Editor.md

```diff
@@ -9,3 +9,58 @@ There are various videos on the internet documenting the use of the built-in lev
 - [Advanced Tutorial (Part 1/2)](https://www.youtube.com/watch?v=WBdwwcLD-vw) by Oliver Buo
 - [Advanced Tutorial (Part 2/2)](https://www.youtube.com/watch?v=UoaGDuBax6E) by Oliver Buo
 - [Expert Tutorial](https://www.youtube.com/watch?v=lL3oZbPfw08) by Oliver Buo
+
+## Share Levels
+
+When you finished a level or even more, there is an easy way to share it to the SuperTux community.
+
+### Step 1) Go to the SuperTux folder and check if all files are there.
+You can find them at C:\Users\YOUR-USERNAME\AppData\Roaming\SuperTux\supertux2\levels\YOUR-LEVELSETNAME\ on Windows or /home/YOUR-USERNAME/.local/share/supertux2/levels/YOUR-LEVELSETNAME/ on Linux.
+There should be some files called level1.stl, level2.stl and so on, depending on the number of levels you have created. Additionally, there will be a file called worldmap.stwm, if you made a world.
+The last file should be called "info" without any file ending. It contains the following code:
+
+(supertux-level-subset</br>
+(title (_ "Name that should be displayed"))</br>
+(description (_ "A short description"))</br>
+(levelset #f)</br>
+(hide-from-contribs #f)</br>
+)
+
+Title: The name of your add-on that should be displayed.</br>
```

Comment:
- Make this a table, that will look nicer and save you the br-tag hack
- Use `title`, `description`, … as it is code
- Add a remark that `(_ "…")` means it's translatable?
- "…that will be displayed to players"